rfcs/text/0002-typechecking-struct-proper-values.md

```diff
@@ -55,28 +55,28 @@ declaration. For example, here is a struct `Point` which has the
 
 In addition, some struct properties requires a function to take not only the
 associated struct instance, but also other instance created by the same struct
-constructor. To construct appropriate type for such a function, we use `Self` to
+constructor. To construct appropriate type for such a function, we use `Imp` to
 specify the implementor type. For example, here is a type annotation for
-`prop:equal` struct property:
+`prop:equal+hash` struct property:
```

Comment:
I came up with `prop:equal` because `prop:equal+hash` expects a list of 3 functions, which, I think, would make the example a little more complicated. But it looks like `prop:equal` is confusing.  What about adding a footnote to `prop:equal`,  or changing the name to something like `prop:fake-equal`?

Reply:
Ok, I'll go back to `prop:equal` and add a note.

Is there a better example than `prop:equal+hash` to use in the docs for `Imp` ?

Reply:
I haven't found any other built-in properties that rely on `Imp`.